src/main/resources/lib/guillotine/queries/fragments/content-types/overviewFragment.graphql

```diff
@@ -4,6 +4,7 @@
 fragment contentOverviewPage on no_nav_navno_Overview {
     dataAsJson
     data {
+        audience
```

Comment:
Skal denne også ha underkategorier?

Reply:
Overview kan ha flere målgrupper, og bruker derfor ikke mixin for audience.